sender/sql.go

```diff
@@ -125,6 +132,13 @@ func (sq *SQL) init() {
 	sq.prep()
 }
 
+func SQLiteDBExists(file string) bool {
+	if _, err := os.Stat(file); err != nil {
+		return false
+	}
+	return true
+}
```

Comment:
`os.Stat()` can return an error which can be checked by `os.IsNotExist(err)` for file existence. I think this could be useful here.

Furthermore, I think it can be made private as it is only used by this sender, otherwise, it deserves some comments if it is to be public (denoted by the capital first letter of its name).

Other than that, maybe this could be a more general "connection check" than verifying if a file exists (or not), e.g. being able to open the file etc., and maybe it can be executed in `Verify()` as well.